runtime/secrets/secrets.go

```diff
@@ -243,14 +252,6 @@ type SSHAuth struct {
 	Password   string
 }
 
-// GitHubAppAuth holds GitHub App authentication credentials.
-type GitHubAppAuth struct {
-	AppID          string
-	InstallationID string
-	PrivateKey     []byte
-	BaseURL        string
-}
-
```

Comment:
how necessary is it to switch to `map[string]byte` here?
can we update callers to use this more type-safe version?